docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,2 +1,142 @@
+# VIC Model Parameters
+
 !!! Warning
     Docs not written yet.
+
+# Converting from VIC 4 to VIC 5
+
+We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via [`tonic`](https://github.com/UW-Hydro/tonic).
+
+### Installing Tonic Using Anaconda
+
+The easiest way to install Tonic and its dependencies is to use the [Anaconda Python distribution](https://store.continuum.io/cshop/anaconda/).
+
+To install Anaconda, follow these two simple steps (check to make sure the installer version is the most current)
+
+1.  download and run the Anaconda installer:  [http://continuum.io/downloads](http://continuum.io/downloads)
+
+2.  setup a virtual environment for Tonic
+```shell
+conda create -n tonic anaconda
+source activate tonic
+```
+
+*Note:  you'll need to do the `source activate tonic` to activate the Tonic virtual environment in any new shells.*
+
+Now, download the Tonic source code:
+
+```shell
+git clone git@github.com:UW-Hydro/tonic.git
+```
+
+From the Tonic source code repository, Tonic can be installed using Python's `distutils`:
```

Comment:
You might want to tell the user explicitly that a "tonic" subdirectory was created and that they should cd to it.  Even though it might seem like overkill or that the user ought to understand this, some people will have questions, there's always someone who interprets things to mean something else, etc.  Listing the exact steps to take, down to the last detail, will help prevent any confusion.